src/pages/OrganizationProjectsPage/OrganizationProjectsPage.tsx

```diff
@@ -166,7 +166,7 @@ const ProjectItem = ({
         <div className="org">
           <Link to={to}>
             <h3>
-              {title}{' '}
```

Comment:
Are you sure we don't want to add a space between `title` and the `DeprecatedIcon`?